### PR TITLE
add oop_03 task

### PR DESCRIPTION
**Pull request by wilden5**
Hello, main store functionality and reflection was added

### CODE REVIEW

File: domain/src/main/java/com/denis/domain/Category.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package com.denis.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Category {
+    public String name;
```
Comment (artemmoroz) [inline domain/src/main/java/com/denis/domain/Category.java]: i think this field should be private or protected

File: domain/src/main/java/com/denis/domain/Category.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    private List<Product> productList;
```
Comment (artemmoroz) [inline domain/src/main/java/com/denis/domain/Category.java]: this seems to be protected

File: store/src/main/java/com/denis/store/Store.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        try {
+            this.categoryList = new RandomStorePopulator().getRandomCategory();
+        } catch (Exception exception) {
+            System.out.println("ERROR: RandomStorePopulator = " + exception.getMessage());
+        }
+    }
+
+    @Override
+    public String toString() {
+        List<String> printStore = new ArrayList<>();
```
Comment (artemmoroz) [inline store/src/main/java/com/denis/store/Store.java]: it is better to you StringBuffer or StringBuilder for that.

**Review by artemmoroz — CHANGES_REQUESTED** [review]

**Comment by wilden5** [comment]
Hello, I've changed all issues that you described above

**Review by artemmoroz — APPROVED** [review]

**Pull request merged**